src/proc/pointcloud.h

```diff
@@ -16,17 +16,17 @@ namespace librealsense
 
         const rs2_intrinsics*   _depth_intrinsics_ptr;
         const float*            _depth_units_ptr;
-        const rs2_intrinsics*   _mapped_intrinsics_ptr;
+        const rs2_intrinsics*   _other_intrinsics_ptr;
```

Comment:
Consider using `librealsense::optional_value` instead of a pointer and value type. See `align` for reference